<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>593</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>565</width>
    <height>593</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>565</width>
    <height>593</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame_4">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="logo_utn.qrc">:/new/imagen/UTN_Parana.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_6">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Ingenieria Electrónica - Proyecto Final</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Carga Programable DC - Gotte A, José A</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Status:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_status">
              <property name="text">
               <string>No Detectado</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="lineEdit_input_val">
           <property name="text">
            <string>0</string>
           </property>
           <property name="maxLength">
            <number>5</number>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_input_val">
           <property name="text">
            <string>[A]</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_modo">
           <item>
            <property name="text">
             <string>CC</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>CP</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>CR</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="lineEdit_input_time">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_time_unit">
           <item>
            <property name="text">
             <string>[seg]</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>[min]</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="pushButton_add">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_del">
           <property name="text">
            <string>Del</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="rowCount">
          <number>0</number>
         </property>
         <column>
          <property name="text">
           <string>Mode</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Input</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Unit</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Time</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Unit</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QFrame" name="frame_3">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QPushButton" name="pushButton_delete_all">
         <property name="text">
          <string>Delete All</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Duracion:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>00:00</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_almacenar">
         <property name="text">
          <string>Almacenar Datos</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_run">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Run</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_stop">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="logo_utn.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>comboBox_modo</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>slot_Update_unit(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_add</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>slot_add()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>493</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_input_val</sender>
   <signal>editingFinished()</signal>
   <receiver>MainWindow</receiver>
   <slot>slot_validate_val()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_input_time</sender>
   <signal>editingFinished()</signal>
   <receiver>MainWindow</receiver>
   <slot>slot_validate_time()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>255</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_modo</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>slot_validate_val()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_del</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>slot_delete()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>504</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableWidget</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>MainWindow</receiver>
   <slot>slot_cambio_select()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>282</x>
     <y>340</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_delete_all</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>slot_delete_all()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>539</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_run</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>slot_run()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>419</x>
     <y>539</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_stop</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>slot_stop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>505</x>
     <y>539</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_Update_unit(int)</slot>
  <slot>slot_start_running()</slot>
  <slot>slot_add()</slot>
  <slot>slot_validate_val()</slot>
  <slot>slot_validate_time()</slot>
  <slot>slot_delete()</slot>
  <slot>slot_cambio_select()</slot>
  <slot>slot_delete_all()</slot>
  <slot>slot_run()</slot>
  <slot>slot_stop()</slot>
 </slots>
</ui>
